ydb/core/viewer/json_pipe_req.cpp

```diff
@@ -748,6 +748,12 @@ void TViewerPipeClient::RequestDone(ui32 requests) {
     }
 }
 
+void TViewerPipeClient::ResetAndBootstrap() {
```

Comment:
why?!

Reply:
Simply calling Bootstrap is not an option, as it resets the Request counter. This manifests in some requests waiting for a timeout (10 seconds) before completing.

Implementing a clean solution for accurate Request counting quickly is challenging. Resetting the counter is the easiest approach.